src/app/messages/en/messages.json

```diff
@@ -1,120 +1,122 @@
-  "i18n:search-results:bar:query:no-results": "Sorry, no results found for ",
-  "i18n:search-results:bar:query:results": "{total, plural, one {search result for } other {search results for }}",
-  "i18n:search-results:bar:search-icon:value": "Search",
-  "i18n:search-results:bar:toggle-text:mobile": "Search results",
-  "i18n:search-results:bar:close-text:mobile": "Close",
-  "i18n:toc:title": "Table of Contents",
-  "i18n:toc:toggle:closed": "Click to open the Table of Contents",
-  "i18n:toc:toggle:opened": "Click to close the Table of Contents",
-  "i18n:toolbar:print:aria-label": "print",
-  "i18n:toolbar:print:text": "Print",
-  "i18n:toolbar:search:placeholder": "Search this book",
-  "i18n:toolbar:search:toggle": "Search this book",
-  "i18n:toolbar:study-guides:text": "Study guides",
-  "i18n:toolbar:highlights:text": "My highlights",
-  "i18n:toolbar:highlights:popup:heading": "My Highlights and Notes",
-  "i18n:toolbar:highlights:popup:close-button:aria-label": "Close My Highlights",
-  "i18n:toolbar:highlights:popup:body:highlights-free": "Log in to highlight. 100% FREE.",
-  "i18n:toolbar:highlights:popup:body:note:highlight": "Highlight",
-  "i18n:toolbar:highlights:popup:body:note:add-notes": "Add notes",
-  "i18n:toolbar:highlights:popup:body:note:study-guide": "Make your study guide",
-  "i18n:toolbar:highlights:popup:body:note:filter-chapters": "Filter by chapters & colors",
-  "i18n:toolbar:highlights:popup:body:note:text": "Note:",
-  "i18n:toolbar:highlights:popup:body:tooltip:review": "Review all your highlights and notes in one convenient place.",
-  "i18n:toolbar:highlights:popup:heading:no-highlights": "No results.",
-  "i18n:toolbar:highlights:popup:heading:no-highlights-tip": "Try selecting different <strong>chapter or color filters</strong> to see different results.",
-  "i18n:toolbar:highlights:popup:body:no-highlights-in-book": "You have no highlights in this book",
-  "i18n:toolbar:highlights:popup:body:add-highlight": "Make a highlight and add a note",
-  "i18n:toolbar:highlights:popup:body:use-this-page": "Then use this page to create your own study guide",
-  "i18n:toolbar:highlights:popup:help-info": "Need help? Visit the <a data-testid='support-link' data-analytics-label='support-link' target=\"_blank\" href=\"https://openstax.secure.force.com/help/articles/FAQ/Using-the-highlighting-and-note-taking-feature-on-mobile-or-tablet?search=highlighting\">Support Center</a>",
-  "i18n:toolbar:highlights:popup:button:back-to-top": "Go back to top",
-  "i18n:toolbar:buy-book:aria-label:text": "Buy book",
-  "i18n:toolbar:buy-book:text": "Order a print copy",
-  "i18n:cta:heading": "Sign up and continue reading. It's 100% free.",
-  "i18n:cta:text": "Read, search, highlight, add notes, build study guides -- all for <strong>free</strong>.",
-  "i18n:cta:button:signup": "Sign up",
-  "i18n:cta:button:separator": "or",
-  "i18n:cta:button:login": "Log in"
-}
+	"i18n:a11y:skipToContent": "Skip to Content",
```

Comment:
you changed this whole file from 2 space to 4 space indentation, please change back